src/app/components/course-item/course-item.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="courseWrapper">
```

Comment:
Please, avoid using camel case in the class name. There a some reason to not use it in class name, following the spec https://www.w3.org/TR/CSS21/syndata.html#characters

Reply:
Also as a good approach, let's use structural directive `*ngIf="course"` to be sure that it will be rendered only if it truthy